### PR TITLE
drivers: imx: enable CFG_TZC380 for imx6ull

### PR DESCRIPTION
**Pull request by andrejro**
Enable the region autoconfiguration for imx6ull by default.

### CODE REVIEW

**Comment by jforissier** [comment]
LGTM, but could someone familiar with imx6ull confirm?

**Comment by Emantor** [comment]
Yes, imx6ull uses a TZC-380 as do all i.MX processors I know of. Only special cases could be the "ulp" variants.

File: core/arch/arm/plat-imx/conf.mk [hunk 1]
```diff
@@ -88,6 +88,7 @@ else ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx6ull-flavorlist)))
 $(call force,CFG_MX6,y)
 $(call force,CFG_MX6ULL,y)
 $(call force,CFG_TEE_CORE_NB_CORE,1)
+$(call force,CFG_TZC380,y)
```
Comment (etienne-lms) [inline core/arch/arm/plat-imx/conf.mk:91]: Should be `CFG_TZC380 ?= y` as per commit log description `... by default.`.

**Review by etienne-lms — CHANGES_REQUESTED** [review]

File: core/arch/arm/plat-imx/conf.mk (hunk 1 above)
Comment (andrejro) [reply core/arch/arm/plat-imx/conf.mk:91]: The `call force` was used to align the `imx6ull` with the rest of `imx6` family.
Giving the security role TZC380 plays in the context of TEE it is preferable to force it to be enabled.

**Comment by jforissier** [comment]
> Giving the security role TZC380 plays in the context of TEE it is preferable to force it to be enabled.

That's fine, in this case please mention "Always enable the region autoconfiguration for imx6ull." in the commit description. With that:
`Acked-by: Jerome Forissier <jerome@forissier.org>`

**Comment by andrejro** [comment]
@jforissier done. Thanks!

**Pull request merged**